src/sound/LICENCE.md

```diff
@@ -0,0 +1,22 @@
+# Sound effect licences
```

Comment:
Note, I've asked the legal team about whether it's acceptable to use these files https://github.com/element-hq/legal-compliance/issues/506.

Reply:
Legal resolved this.